spec/factories/credits.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: credits
+#
+#  id         :integer          not null, primary key
+#  cost       :decimal(, )
+#  order_id   :integer
+#  quote_id   :integer
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+FactoryGirl.define do
+  factory :credit do
+    cost "9.99"
+    order_id 1
```

Comment:
you can have the credit factory generate your order and quote, just change these to:

```ruby
FactoryGirl.define do
  factory :credit do
    cost '9.99'
    order
    quote
  end
end
```

Reply:
These are getting errors with the tests when changing it.

Reply:
<img width="617" alt="screen shot 2017-09-01 at 11 21 13 am" src="https://user-images.githubusercontent.com/16567429/29978865-b24c0d6e-8f08-11e7-949d-4b2e740f1d9e.png">